keymaker/report.py

```diff
@@ -0,0 +1,20 @@
+from pydriller.metrics.process.change_set import ChangeSet
+
+
+class ReportGenerator:
+    def __init__(self):
+      self.watchlist = []
+      self.stats = {}
```

Comment:
I'd imagine we'd pack all the metrics for the report in this data structure. For now kept it a `dict` in the future, we can make it a class as we understand our requirements.